tensorflow/lite/micro/kernels/detection_postprocess.cc

```diff
@@ -0,0 +1,857 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include <numeric>
+
+#define FLATBUFFERS_LOCALE_INDEPENDENT 0
+#include "flatbuffers/flexbuffers.h"  // from @flatbuffers
```

Comment:
`third_party/tensorflow/lite/micro/kernels/detection_postprocess.cc:18:10: error: module //third_party/tensorflow/lite/micro/kernels:micro_ops does not depend on a module exporting 'third_party/flatbuffers/include/flatbuffers/flexbuffers.h'
see http://go/cpp-features#layering_check; to fix run:
build_cleaner //third_party/tensorflow/lite/micro/kernels:micro_ops
#include "third_party/flatbuffers/include/flatbuffers/flexbuffers.h"`